buildversion/version.go

```diff
@@ -22,3 +22,12 @@ var (
 	BuildTime    = ""
 	BuildCommit  = ""
 )
+
+// PackageManager defines the package manager which was used to install the CLI.
+// You can override this value using -X flag to the compiler ldflags. This is
+// overridden when we build for Homebrew.
```

Comment:
So this still needs done so that `homebrew` will be set here??

Reply:
> So this still needs done so that `homebrew` will be set here??

Err, um, yes? This falls into the bucket of "I still don't know exactly how that will all work". :)